### DISCUSSION TITLE
College is not providing the Student email Address

### DISCUSSION THREAD

**Discussion by Jenis07** [community/community#108411] [GitHub Education]
### Select Topic Area

Question

### Body

So what i have to do now?
i have discuss with sir but they are not providing me this email address.
College Name: Shantilal Shah Engineering College, Bhavnagar Gujarat, India

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @Jenis07 , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Reply by Jenis07** [reply]
Thanks..
